### PR TITLE
Cleanup READMEs for CDAP 3.0 release

### PR DESCRIPTION
**Pull request by awholegunch**
Fixes, tweaks, corrections, adjustments. Each file has the same copyright statement at the end. When the file is included elsewhere, the include will stop before it.

### CODE REVIEW

File: cdap-file-drop-zone/README.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+    :license: See LICENSE file in this repository
+
```
Comment (anew) [inline cdap-file-drop-zone/README.rst:5]: Copyright is 2014-2015, right? Because these were written in 2014...

**Comment by awholegunch** [comment]
Fixed copyright.

File: cdap-stream-clients/java/README.rst [hunk 2]
```diff
@@ -60,7 +65,7 @@ Example::
 
 Creating a Stream
 -----------------
-Create a new Stream with the *stream id* "streamName"::
+Create a new Stream with the *stream-id* "streamName"::
```
Comment (anew) [inline cdap-stream-clients/java/README.rst]: Create a new Stream named `streamName` ?

Comment (anew) [inline cdap-stream-clients/java/README.rst]: Hmm not sure what the formatting should be... what is the convention elsewhere? What is italic used for in other places?

**Comment by anew** [comment]
LGTM

File: cdap-stream-clients/java/README.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-stream-clients/java/README.rst]: it's probably backward. Names are in italics. Parameters are in between angle brackets, and in literal. Let me fix these...

**Pull request merged**